src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -46,7 +46,8 @@ public LightingManager(ICubicWorld world) {
 
 		this.skylightCubeDiffuseProcessor = new SkyLightCubeDiffuseProcessor(world, "Sky Light Diffuse", 5);
 
-		this.firstLightProcessor = new FirstLightProcessor("First Light", world.getCubeCache(), 1);
+		// TODO: Setting GeneratorStage to null. Analyze effects.
+		this.firstLightProcessor = new FirstLightProcessor(null, "First Light", world.getCubeCache(), 1);
```

Comment:
Right now you pretty much  disabled the whole "checking if lighting can be safely calculated" thing. But this also makes the change in isAreaLoaded "work".